solr/core/src/java/org/apache/solr/core/FileSystemConfigSetService.java

```diff
@@ -250,7 +251,7 @@ public FileVisitResult visitFile(Path file, BasicFileAttributes attrs)
           public FileVisitResult postVisitDirectory(Path dir, IOException ioException) {
             String relativePath = configDir.relativize(dir).toString();
             if (!relativePath.isEmpty()) {
-              filePaths.add(relativePath + "/");
+              filePaths.add(normalizePathToForwardSlash(relativePath + File.separator));
```

Comment:
shouldn't this always end in '/' ?   If we agree on this, then there's a gap in the tests.

Reply:
yea I can fix this.

Reply:
So this was normalizing to `/` just semi convolutedly. Fixed in 836f33ad38a2bcca839e37d63ff35a78acbee404 and added a comment there to clarify.